src/main/java/Bert.java

```diff
@@ -0,0 +1,354 @@
+                "____________________________________________________________\n"
+        );
+    }
+
+    /**
+     * Saves the task list into ip/data/bert.txt.
+     */
+    private static void saveTasks() {
+        try {
+            ensureTaskFileExists();
+        } catch (IOException e) {
+            System.out.println(
+                    "____________________________________________________________\n" +
+                    "OOPS!!! An error occurred while creating the task file.\n" +
+                    "____________________________________________________________\n"
+            );
+            return;
+        }
+
+        try {
+            Bert.writeToFile(FILE_PATH, Bert.taskListToSaveFormat());
+        } catch (IOException e) {
+            System.out.println(
+                    "____________________________________________________________\n" +
+                    "OOPS!!! An error occurred while saving tasks.\n" +
+                    "____________________________________________________________\n"
+            );
+        }
+    }
+
+    /**
+     * Checks if ip/data/bert.txt exists.
+     * If the directory or the file does not exist,
+     * creates the directory and the file.
+     *
+     * @throws IOException This exception is thrown when an error occurs while creating
+     *          the file.
+     */
+    private static void ensureTaskFileExists() throws IOException {
+        File file = new File(FILE_PATH);
+        if (!file.getParentFile().isDirectory()) {
+            file.getParentFile().mkdir();
+        }
+        if (!file.exists()) {
+            file.createNewFile();
+        }
+    }
+
+    /**
+     * Converts the task list into its save format.
+     *
+     * @return The String representation of the formatted task list
+     */
+    private static String taskListToSaveFormat() {
+        StringBuilder sb = new StringBuilder();
+        for (Task t : al) {
+            sb.append(t.toSaveFormat() + System.lineSeparator());
+        }
+        return sb.toString();
+    }
+
+    /**
+     * Writes the textToAdd input into a file specified by filePath.
+     *
+     * @param filePath The path to the file to be written
+     * @param textToAdd The String of text to be written in the file
+     * @throws IOException This exception is thrown when an error occurs while opening
+     *          the file.
+     */
+    private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+    /**
+     * Loads tasks from the save data into the chatbot.
+     */
+    private static void loadTasks() {
+        try {
+            ensureTaskFileExists();
+            File file = new File(FILE_PATH);
+            Scanner sc = new Scanner(file);
+            while (sc.hasNext()) {
+                String task = sc.nextLine();
+                switch (task.charAt(0)) {
```

Comment:
You can follow the [Java Coding Standard](https://se-education.org/guides/conventions/java/intermediate.html#:~:text=The%20switch%20statement%20should%20have%20the%20following%20form%3A) for the switch statement.